contracts/test/allowlist.test.ts

```diff
@@ -95,6 +95,7 @@ describe("AllowlistPaymaster", function () {
   it("should prevent non-owners from withdrawing funds", async function () {
     try {
       await paymaster.connect(userWallet).withdraw(userWallet.address);
+      throw new Error("Should not be able to withdraw funds");
```

Comment:
This test case purpose is to throw an error. See the expect statement in the catch block.

Reply:
If the error is not thrown in line 97, does the program not continue within the try statement and the test passes without that being the intended outcome?

Reply:
The purpose of the test case is to ensure that when a non-owner wallet attempts to call the `withdraw` function it will fail with the expected error. So in this test case it will never not throw an error on line 97, as the userWallet attempting to call `withdraw` is not the owner which is why this test passes with the expect statement in the catch block.